clip_image_classifier.py

```diff
@@ -47,23 +48,28 @@ def __init__(
 
     @requests
     def classify(self, docs: DocumentArray, parameters: dict, **kwargs):
-
+        classes = parameters.get('classes',self.classes)
+        if classes == None :
+            raise ValueError(
+                '``classes`` parameter is mandatory. Pass it in parameters'
+                'or in constructor. e.g. ["This is a cat","This is a person", "This is a dog"]'
+            )
         for docs_batch in docs.traverse_flat(parameters.get('traversal_paths', self.traversal_paths)).batch(
             batch_size=parameters.get('batch_size', self.batch_size)
         ):
             image_batch = docs_batch.get_attributes('blob') 
             with torch.inference_mode():
-                input = self._generate_input_features(image_batch)
+                input = self._generate_input_features(classes,image_batch)
```

Comment:
So there should be a space after the comma - please format everything using `black`. Check out the pre-commit hooks from some other executor repos